staging/src/k8s.io/apiserver/pkg/server/options/server_run_options.go

```diff
@@ -245,5 +252,10 @@ func (s *ServerRunOptions) AddUniversalFlags(fs *pflag.FlagSet) {
 		"will return success, but /readyz immediately returns failure. Graceful termination starts after this delay "+
 		"has elapsed. This can be used to allow load balancer to stop sending traffic to this server.")
 
+	fs.BoolVar(&s.RetryWhenHasNotBeenReady, "retry-when-not-ready", s.RetryWhenHasNotBeenReady, ""+
```

Comment:
retry sounds like kube-apiserver retries something. `--retry-after-429-when-not-ready`.

Reply:
am open for better suggestions